presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -1266,4 +1267,26 @@ private void assertGeomFromBinary(String wkt)
     {
         assertFunction(format("ST_AsText(ST_GeomFromBinary(ST_AsBinary(ST_GeometryFromText('%s'))))", wkt), VARCHAR, wkt);
     }
+
+    @Test
+    public void testGeometryJsonConversion() throws Exception {
```

Comment:
Please add tests for:
1. Empty Geometries (of all types)
2. Deserializing valid GeoJSON strings
3. Deserializing invalid GeoJSON strings

Reply:
3. Should include valid JSON but invalid GeoJSON, as well as invalid JSON.